docs/tutorials/01_neural_networks.ipynb

```diff
@@ -13,9 +13,8 @@
     "The following different available neural networks will now be discussed in more detail:\n",
     "\n",
     "1. `NeuralNetwork`: The interface for neural networks.\n",
-    "2. `OpflowQNN`: A network based on the evaluation of quantum mechanical observables.\n",
-    "3. `TwoLayerQNN`: A special `OpflowQNN` implementation for convenience. \n",
-    "3. `CircuitQNN`: A network based on the samples resulting from measuring a quantum circuit."
+    "2. `EstimatorQNN`: A network based on the evaluation of quantum mechanical observables.\n",
+    "3. `SamplerQNN`: A network based on the samples resulting from measuring a quantum circuit.\n"
```

Comment:
Since this is basics.... I see the SamplerQNN below shows it being created with a Sampler i.e. `sampler=Sampler()`, but estimator is left to default and you don't even see its configured with one. Would it make sense for a sentence or two here to say that these take primitives, BaseEstimator and BaseSampler types respectively, where you can pick reference, runtime, from Aer provider etc so you choose where/how it runs via the primitive and its configuration you make.

Reply:
The assignment `sampler=Sampler()` is left from the time when the parameter was required. I was updating the tutorials in the same the networks were being developed. I removed the `sampler`.

In the beginning I added:

`Each implementation, EstimatorQNN and SamplerQNN, takes in an optional instance of the corresponding Qiskit primitive, namely BaseEstimator and BaseSampler. The latter two define two interfaces of the primitives. Qiskit provides the reference implementation as well as a backend-based implementation of the primitives. The primitives is a frontend to either a simulator or a real quantum hardware. By default, if no instance is passed to a network, an instance of the Qiskit reference primitive is created automatically by the network. For more information about primitives please refer to [Qiskit primitives](https://qiskit.org/documentation/apidoc/primitives.html).`

And before we create a network:

`We don't set the estimator parameter, the network will create an instance of the reference Estimator primitive for us.`

Feel free to edit.